executor/executor_test.go

```diff
@@ -6155,6 +6178,10 @@ func (s *testRecoverTable) TestFlashbackTable(c *C) {
 	tk.MustExec("insert into t values (1),(2)")
 	tk.MustExec("drop table t")
 	tk.MustExec("flashback table t")
+	// Test for flashback database.
+	tk.MustExec("drop database Test2")
+	tk.MustExec("flashback database Test2")
+	tk.MustExec("use Test2")
```

Comment:
Add test case about `flashback database` and the database contains more than 1 tables.